main.js

```diff
@@ -0,0 +1,42 @@
+
+    const slides = [
+    { link: 'careershowtalk.html', title: 'Career Show Talk' },
+    {
+        link: 'web-dev-basics',
+        title: 'Web Development Basics (Urdu)',
+    },
+    {
+        link: 'you-dont-see-me-as-a-dev.html',
+        title: 'I am a Developer, you don\'t see me',
+    },
+    {
+        link: 'optimizing-angular-apps-like-a-pro.html',
+        title: 'Optimizing an Angular App like PRO',
+    },
+    {
+        link: 'angular-schematics-a-time-well-spent.html',
+        title: 'Angular Schematics - A time well spent',
+    },
+    {
+        link: 'wisdom-perfection-and-crafting-excellence-at-work.html',
+        title: 'Wisdom, Perfection, and crafting Excellence at Work',
+    },
+    {
+        link: 'ui-for-backenders.html',
+        title: 'UI for Backenders',
+    }
+    ];
+    const grid = document.getElementById('talksGrid');
```

Comment:
@jenster5 this file doesn't need to be formatted properly. Can you please format it?

Reply:
applied prettier here